package.json

```diff
@@ -61,7 +61,11 @@
 	},
 	"publishConfig": {
 		"ignore": [
-			".github/workflows"
+			".github/workflows",
+			"example",
+			".eslintrc",
+			".nycrc",
```

Comment:
i forget whether this file is required for tests to pass; you can check that locally by temporarily deleting it.

Reply:
Right - the command is `npm explore foo && npm install && npm test`, which can be installed from a local cache or registry.